packages/nightingale-sunburst/src/nightingale-sunburst.ts

```diff
@@ -0,0 +1,426 @@
+import { LitElement, html, PropertyValues } from "lit";
+import { property, customElement } from "lit/decorators.js";
+import {
+  select,
+  partition as d3partition,
+  hierarchy,
+  scaleOrdinal,
+  quantize,
+  interpolateRainbow,
+  HierarchyRectangularNode,
+} from "d3";
+
+const superkingdoms = ["bacteria", "viruses", "archaea", "eukaryota", null];
```

Comment:
So that list is mostly used to assign a color to each kingdom. Having a null there gives the same color to anything that doesn't belong to any of the other kingdoms.